airbyte-integrations/bases/base-normalization/Dockerfile

```diff
@@ -30,5 +30,5 @@ ENTRYPOINT ["/airbyte/entrypoint.sh"]
 
 # 0.3.x is tombstoned.
 # The next minor bump should go directly to 0.4.0
-LABEL io.airbyte.version=0.2.25
+LABEL io.airbyte.version=strict_comparison
```

Comment:
is the idea behind this label that when we are ready to roll this out to 100% of workspaces, we will change "strict_comparison" label to the next version (0.2.26 or whatever it happens to be) and remove the feature flags?

Reply:
exactly - bump to new version + update destination_definitions for global rollout.